### PR TITLE
WBCAMS-637 quiz validation error messages

### PR DESCRIPTION
**Pull request by rjkuyvenhoven**
Work-Preferences-Quiz-Career-Discovery-Quizzes-11-13-2024_09_10_AM

### CODE REVIEW

File: src/web/modules/custom/workbc_cdq_career_match/js/workbc-cdq-career-match.js [hunk 1]
```diff
@@ -303,4 +303,28 @@
     }
   };
 
+
+  Drupal.behaviors.CareerQuizzes = {
+    attach: function (context, settings) {
+
+      $("#block-workbc-cdq-content > form > div.form-wrapper").each(
+        function () {
```
Comment (infojunkie) [inline src/web/modules/custom/workbc_cdq_career_match/js/workbc-cdq-career-match.js:311]: Just to confirm: The logic of injecting the `error` class is done in the core webform module? So all we need to do here is style it differently.

**Review by infojunkie — COMMENTED** [review]
Just a question to confirm before I approve.

File: src/web/modules/custom/workbc_cdq_career_match/js/workbc-cdq-career-match.js (hunk 1 above)
Comment (rjkuyvenhoven) [reply src/web/modules/custom/workbc_cdq_career_match/js/workbc-cdq-career-match.js:311]: yes

**Review by infojunkie — APPROVED** [review]
Looks good, thanks

**Pull request merged**